packages/components/src/guide/README.md

```diff
@@ -71,3 +57,19 @@ Use this to customize the label of the _Finish_ button shown at the end of the g
 
 -   Type: `string`
 -   Required: No
+-	Default: `'Finish'`
+
+### onFinish
+
+A function which is called when the guide is finished. The guide is finished when the modal is closed or when the user clicks _Finish_ on the last page of the guide.
+
+-   Type: `( event?: KeyboardEvent< HTMLDivElement > | SyntheticEvent ) => void`
+-   Required: Yes
+
+### pages
+
+A list of objects describing each page in the guide. Each object **must** contain a `'content'` property and may optionally contain a `'image'` property.
```

Comment:
The prop descriptions in the README should be the same as in the types JSDocs — we could probably keep the ones in the README as source of truth, and update the JSDocs in the types.ts file?

Reply:
Fixed via https://github.com/WordPress/gutenberg/pull/47493/commits/bbf3264c2387a9a19a9e74cedda1a04d2b01f680